lib/browser.js

```diff
@@ -18,3 +18,4 @@ require('./http/xhr');
 
 if (typeof window !== 'undefined') window.AWS = AWS;
 if (typeof module !== 'undefined') module.exports = AWS;
+if (typeof self !== 'undefined')module.exports = AWS; 
```

Comment:
@cesarpachon 
It looks like the tests are failing on the linter because there isn't a space before `module`. Do you mind updating that?